test/tools/jsontests/LegacyTests.h

```diff
@@ -37,6 +37,30 @@ class BCGeneralStateTestsSuiteLegacyConstantinople : public BCGeneralStateTestsS
     }
 };
 
+class BlockchainInvalidTestSuiteLegacyConstantinople : public BlockchainInvalidTestSuite
```

Comment:
(I would name these classes rather like `LegacyConstantinopleBlockchainInvalidTestSuite` - it's easier to see what it is derived from by the suffix, it's a common convention like `class Thing  <- class DerivedThing`